frontend/packages/component-library/src/video/Video.tsx

```diff
@@ -0,0 +1,69 @@
+import classNames from 'classnames';
+import {HTMLAttributes} from 'react';
+
+import moduleStyles from './video.module.scss';
+import LinkButton from '../button/LinkButton';
+
+export interface VideoProps extends HTMLAttributes<HTMLElement> {
+  /** Video Title */
+  videoTitle?: string;
+  /** Video YouTube ID */
+  youTubeId?: string;
+  /** Video fallback */
+  videoFallback?: string;
+  /** Show caption */
+  showCaption?: boolean;
+  /** Video custom className */
+  className?: string;
+}
+
+/**
+ * ### Production-ready Checklist:
+ * * (✘) implementation of component approved by design team;
+ * * (✘) has storybook, covered with stories and documentation;
+ * * (✘) has tests: test every prop, every state and every interaction that's js related;
+ * * (see ./__tests__/Video.test.tsx)
+ * * (✘) passes accessibility checks;
+ *
+ * ###  Status: ```WIP```
+ *
+ * Design System: Video Component.
+ */
+
+export const Video: React.FC<VideoProps> = ({
+  youTubeId,
+  videoTitle,
+  videoFallback,
+  showCaption,
+  className,
+}: VideoProps) => (
+  <figure className={moduleStyles.video}>
+    <div className={moduleStyles.videoWrapper}>
+      <iframe
```

Comment:
In the case `youTubeId` is undefined, it shows a YouTube video but shows an error screen. I wonder how we should handle if this prop is undefined since when the component is first dragged in, it will be undefined (unless we define a default value)

Reply:
This is a required field on the Content Type in Contentful so I don't think it'll be a problem there. Should we make it required in the component itself too?